pkg/graph/resolvers/model_plan_phase.go

```diff
@@ -231,7 +301,7 @@ func ConstructPhaseSuggestionEmailTemplates(
 	return emailSubject, emailBody, nil
 }
 
-func TrySendEmailForPhaseSuggestionByModelPlanID(
+func TryNotificationSend(
```

Comment:
This names is probably a little too vague since it is for a specific notification type (and is scoped to the resolvers package)

Reply:
that's fair, switching to TryNotificationSendIncorrectModelStatus